spartan/aztec-network/values/alpha-testnet-bots.yaml

```diff
@@ -0,0 +1,14 @@
+bot:
```

Comment:
Deploying this will deploy an entire stack no? since the default values.yaml includes things with non zero replicas?